### DISCUSSION TITLE
Automatic Login?

### DISCUSSION THREAD

**Discussion by ByteFetch** [tteck/Proxmox#140] [Q&A] [answered]
Hi,

Is there somewhere that further explains the pros/cons of choosing between an automatic login or a password?

**Comment by al13nus** [comment]
Would be good to know how to deactive it afterwards.

**Comment by tteck** [comment]
~Once a LXC is created with automatic login, there is no reversal.~
https://github.com/tteck/Proxmox/discussions/324

**Reply by FrankDUS69** [reply]
I created a docker LXC using your script with default settings, but the automatic login doesn´t work. Any suggestion? It´s not possible to remove the LXC actually.

**Reply by tteck** [reply]
Sounds like the script didn't finish. You must stay on the shell until you see `✓ Completed Successfully!`

**Reply by FrankDUS69** [reply]
Maybe it was my mistake. But how can I remove the LXC from PVE shell? Or is there another way to reactivate the script to finish?

**Reply by tteck** [reply]
Stop the LXC and select remove

**Reply by tteck** [reply]
![Screenshot 2022-11-03 11 09 30 AM](https://user-images.githubusercontent.com/21342670/199759262-e3fb0d2a-6ba9-48ef-8306-314f2e52adb9.png)

**Reply by FrankDUS69** [reply]
It was greyed out. but after reboot of Proxmox it isavailable. I think I need more patience sometimes.., TY.

**Comment by tteck** [comment]
You can set a root password (`sudo passwd root`) but, it will still automatically login.

**Reply by al13nus** [reply]
Is this atuomatical login a secure thing? That the only concerne I have because I will create anyway another user on the LXC for SSH.

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/discussions/324
